### PR TITLE
Add pydstool

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
@isuruf 

I did not know what to do with the sha256

pydstool information:

https://github.com/robclewley/pydstool
https://pypi.python.org/pypi/PyDSTool/0.90.2

### CODE REVIEW

File: recipes/pydstool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{% set version = "0.90.2" %}
+{% set sha256 = "d2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
+package:
+  name: pydstool
+  version: {{ version }}
+
+source:
+  fn: pydstool-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/c8/5d/9618268c502d6b38d7cfe9be7e9ee967e77c0416c1660c8dbb0368c8ce29/PyDSTool-{{ version }}.tar.gz
```
Comment (isuruf) [inline recipes/pydstool/meta.yaml]: Change this to `https://pypi.io/packages/source/P/PyDSTool/PyDSTool-{{ version }}.tar.gz`

**Comment by ChrisRackauckas** [comment]
Not sure what to do about that linting issue. I can't find details on the error.

File: recipes/pydstool/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{% set version = "0.90.2" %}
+{% set sha256 = "d2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
```
Comment (isuruf) [inline recipes/pydstool/meta.yaml]: Change this to `6e0cbceab8088ee8cbd0839202cdbaeebbe4ab2e6d1a0d0053c7de30268587c1`

Got it from running `sha256sum` on the downloaded archive.

File: recipes/pydstool/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - PyDSTool.Toolbox.optimizers.line_search
+    - PyDSTool.Toolbox.optimizers.optimizer
+    - PyDSTool.Toolbox.optimizers.step
+    - PyDSTool.core
+    - PyDSTool.core.codegenerators
+
+requires:
+  - mock
+  - pytest
+  - pytest-xdist
```
Comment (isuruf) [inline recipes/pydstool/meta.yaml]: You can remove the above 4 lines as we are not running the test suite here.

File: recipes/pydstool/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    - numpy >=1.6
+
+  run:
+    - python
+    - six
+    - scipy >=0.9
+    - numpy >=1.6
+
+test:
+  # Python imports
```
Comment (isuruf) [inline recipes/pydstool/meta.yaml]: Remove this comment

File: recipes/pydstool/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    - PyDSTool.core.codegenerators
+
+requires:
+  - mock
+  - pytest
+  - pytest-xdist
+
+about:
+  home: http://www.ni.gsu.edu/~rclewley/PyDSTool/FrontPage.html
+  license: BSD
```
Comment (isuruf) [inline recipes/pydstool/meta.yaml]: Change this to
```
license: BSD 3-Clause
license_file: LICENSE.txt
```

**Comment by ChrisRackauckas** [comment]
Did I do something wrong here? I am not sure why this isn't getting merged.

File: recipes/pydstool/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+{% set version = "0.90.2" %}
+{% set sha256 = "6e0cbceab8088ee8cbd0839202cdbaeebbe4ab2e6d1a0d0053c7de30268587c1" %}
+package:
```
Comment (ocefpaf) [inline recipes/pydstool/meta.yaml]: Can you add a space here? (It can be in the feedstock, this does not hold off merging.)

**Comment by ocefpaf** [comment]
> Did I do something wrong here? I am not sure why this isn't getting merged.

We are just a few reviewers with day jobs. You did nothing wrong :wink: 

Thanks @isuruf for the review!

**Comment by ChrisRackauckas** [comment]
Not sure what the feedstocks are, but I edited the branch I did the commit on to have the space. Do I PR again?

**Comment by ocefpaf** [comment]
> Not sure what the feedstocks are, but I edited the branch I did the commit on to have the space. Do I PR again?

staged-recipes is a place to submit the recipes. Once merged they will become a feedstock, or its own repository where the maintainer will have merge rights. You'll need to wait for the feedstock to be created (it may take a while).

No need for another PR here. Meanwhile take a look at http://conda-forge.github.io/docs/ and let me know if you have any questions about conda-forge.

**Comment by isuruf** [comment]
@ChrisRackauckas , welcome to conda-forge. This recipe will be moved to a new repo https://github.com/conda-forge/pydstool-feedstock in about a day. After the repo is created only the conda packages will be available. You will be asked if you would like to join conda-forge. If you accept, you'll be given commit privileges and access to the CI of that repo (pydstool-feedstock).

One important thing to note is that if you want to make a change, send a PR from a fork. Branches of the main repo are used for maintaining different versions only.

**Comment by ocefpaf** [comment]
@isuruf I guess it is about time we summarize messages like https://github.com/conda-forge/staged-recipes/pull/2752#issuecomment-293871292 in a "Welcome to conda-forge" docs page. Do you want to do it?

**Comment by isuruf** [comment]
Sure

**Pull request merged**